File: Assignment_39/program1.cpp
// Write generic program which accept one value and one number from user. Print that value that number of times on screen.
/*  
    Input : M 7
    Output :M M M M M M M 

    Input : 11 3
    Output : 11 11 11

*/


///////////////////////////////////////////////////////////////////////////////
// File Name   :  program1.cpp
// Description :  Generic program which accept one value and one number from user. Print that value that number of times on screen.
// Autor       :  Ankita Anil Patil
// Date        :  5/07/2025
///////////////////////////////////////////////////////////////////////////////


#include<iostream>
using namespace std;

template <class T>

void Display(T value, int size)
{
    int i = 0; 

    for(i = 0; i < size; i++)
    {
        cout<<value<<"\t";
    }

    cout<<"\n";

}

int main()
{
    Display('M',7);
    Display(11,3);
    Display(3.7,6);
    return 0;
}

File: Assignment_54/program1.java
//  Write java program which accept array from user and reverse each number of that array. 
/*
    Input :  89 687 56 549 87 9 
    Output : 98 786 65 945 78 9 
 */

////////////////////////////////////////////////////////////////////////////////
// File Name   :  program1.java
// Description :  A ccept array from user and reverse each number of that array.          
// Autor       :  Ankita Anil Patil
// Date        :  13/09/2025
///////////////////////////////////////////////////////////////////////////////


import java.util.*;

class MyArray 
{ 
        public void ReverseArray(int Arr[]) 
        {   
            
            int i = 0, iRev = 0,iDigit = 0;
           
            for(i = 0; i < Arr.length; i++)
            {   
                
                while(Arr[i] != 0)
                {
                    iDigit = Arr[i] % 10;
                    iRev = iRev * 10 + iDigit;
                    Arr[i] = Arr[i] / 10;
                }

                Arr[i] = iRev;
                iRev = 0;
                
            }

            System.out.println("After reverse elements in array are : ");
            for(i = 0; i < Arr.length; i++ )
            {
                System.out.print(Arr[i] + " ");
            }
        } 
} 
public class program1
    
{   
    public static void main(String[] args) {
        

    Scanner sobj = new Scanner(System.in);

    System.out.println("Enter size of array");
    int iSize = sobj.nextInt();


    int[] Arr = new int[iSize];
    System.out.println("Enter array elements : ");

    int i = 0;
    
    for(i = 0; i < Arr.length; i++)
    { 
       Arr[i] = sobj.nextInt();
    }

    MyArray mobj = new MyArray();
    mobj.ReverseArray(Arr);

   
    
}
}
